server/repository/src/migrations/v2_01_00/assets/asset_property.rs

```diff
@@ -0,0 +1,40 @@
+use crate::{
+    migrations::{sql, JSON},
+    StorageConnection,
+};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    const PROPERTY_VALUE_TYPE: &str = if cfg!(feature = "postgres") {
+        "PROPERTY_VALUE_TYPE" // This is created as part of the asset_catalogue_property migration
```

Comment:
This probably doesn't matter.. but for completeness
```suggestion
        "property_value_type" // This is created as part of the asset_catalogue_property migration
```